ansible/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Docker YUM depencies present
+  tags:
+    - docker
+  yum:
+    name: ['yum-utils', 'device-mapper-persistent-data', 'lvm2']
+    state: present
+
+- name: Docker YUM repo present
+  tags:
+   - docker
+  yum_repository:
+    name: docker-ce
+    description: Docker-ce YUM repo
+    baseurl: https://download.docker.com/linux/centos/7/x86_64/stable
+
+- name: Docker present
+  tags:
+    - docker
+  yum:
+    name: docker-ce
+    state: present
+
+- name: Docker started
+  tags:
+    - docker
+  service:
+    name: docker
+    state: started
+
+- name: User in Docker group
+  tags:
+    - docker
+  user:
+    name: root
+    groups: docker
```

Comment:
Add local default user to docker group as well as root user.

Reply:
Done